src/pepr/policies/networking.ts

```diff
@@ -16,7 +17,7 @@ import { exemptHostNamespaces } from "./exemptions/networking";
 When(a.Pod)
   .IsCreatedOrUpdated()
   .Validate(request => {
-    if (exemptHostNamespaces(request)) {
+    if (isExempt(Policy.DisallowPrivileged, request)) {
```

Comment:
This looks like the wrong policy? Should be`DisallowHostNamespaces`:
```suggestion
    if (isExempt(Policy.DisallowHostNamespaces, request)) {
```

Reply:
ohp. thanks for catching that.